docs/modules/operation/pages/deep-dive/notifications/strategies/mattermost.adoc

```diff
@@ -22,57 +25,50 @@ First, add the following bit of XML to the `notificationCommands.xml` configurat
+Create `mattermost.properties` in your `opennms.properties.d` directory, and insert the following property:
 
 [source, properties]
-----
 org.opennms.netmgt.notifd.mattermost.webhookURL=https://mattermost.example.com/hooks/bf980352b5f7232efe721dbf0626bee1
-----
 
-Restart OpenNMS so that the `mattermost.properties` file will be loaded. Your new `mattermost` notification command is now available for use in a destination path.
+Customize its values as appropriate for your organization.
+
+{page-component-title} loads `mattermost.properties` upon restart.
+After it is loaded, your new `mattermost` notification command will be available for use in a destination path.
 
-== Additional options
-The following table lists optional properties that you may use in `mattermost.properties` to customize your Mattermost notifications.
+== Additional configuration options
 
-IMPORTANT: To improve the layout, the property names have been shortened to their final component; you must prepend `org.opennms.netmgt.notifd.mattermost.` when using them.
+The following table lists optional properties that you may use in `mattermost.properties` to customize your Mattermost notifications:
 
-.Additional optional parameters for the Mattermost notification strategy
-[options="header"]
-[cols="1,3,1,2"]
+[cols="2,2,1,1"]
 |===
-| Parameter
-| Description
-| Default
-| Example
+| Parameter | Description | Default Value | Example
 
-| channel
+| org.opennms.netmgt.notifd.mattermost.channel
 | Specify a channel or private group other than the one targeted by the webhook.
-| Webhook default
+| _Webhook default_
 | NetOps
 
-| username
+| org.opennms.netmgt.notifd.mattermost.username
 | The username to associate with the notification posts.
-| None
+| _Blank_
 | OpenNMS_Bot
 
-| iconEmoji
+| org.opennms.netmgt.notifd.mattermost.iconEmoji
 | An emoji sequence to use as the icon for the notification posts.
-| No icon
+| _Blank_
 | :metal:
 
-| iconURL
+| org.opennms.netmgt.notifd.mattermost.iconURL
 | The URL of an image to use as the icon for the notification posts.
-| No icon
+| _Blank_
 | \https://example.org/assets/icon.png
 
-| useSystemProxy
+| org.opennms.netmgt.notifd.mattermost.useSystemProxy
 | Should the system-wide proxy settings be used?
 Configure the system proxy settings via <<deep-dive/admin/system-properties/introduction.adoc#ga-opennms-system-properties, system properties>>.
 | true
 | true
 |===
 
-IMPORTANT: Some of the optional configuration parameters are incompatible with some versions of Mattermost.
-           For instance, the `channel` option is known not to work with Mattermost 3.7.0.
-
-For more information on incoming webhooks in Mattermost, see https://docs.mattermost.com/developer/webhooks-incoming.html[Mattermost Integration Guide].
+IMPORTANT: Some of the optional configuration parameters are not compatible with some versions of Mattermost.
+For instance, the `channel` option is known not to work with Mattermost 3.7.0.
```

Comment:
```suggestion

```
Mattermost is up to version 7.6, and 3.7.0 is 6 years old, so we can probably remove this comment.